src/web/dockerflow.rs

```diff
@@ -52,10 +52,7 @@ async fn version() -> HttpResponse {
 /// Returns a status message indicating the current state of the server
 async fn heartbeat(state: web::Data<ServerState>) -> HttpResponse {
     let mut checklist = HashMap::new();
-    checklist.insert(
-        "version".to_owned(),
-        Value::String(env!("CARGO_PKG_VERSION").to_owned()),
-    );
+    checklist.insert("version".to_owned(), Value::String(create_app_version("/")));
```

Comment:
Note that apart from `CARGO_PKG_VERSION`, `CARGO_PKG_NAME` is also included in this field which I assume should be fine to do here.